src/resources/platform-adaptations.md

```diff
@@ -548,6 +552,34 @@ double tap and shows the selection toolbar.
   </div>
 </div>
 
+## UI components
+
+This section includes preliminary recommendations on how to adapt 
+Material widgets to deliver a natural and compelling experience on iOS. 
+Your feedback is welcomed on [issue #8427][]. 
+
+### Alert dialog
+
+Since Android 12, the default UI of alert dialogs 
+(also known as a "basic dialog") follows the design guidelines 
+defined in [Material 3](https://m3.material.io/components/dialogs/overview). 
+On iOS, an equivalent component called “alert” is defined in Apple’s 
+[Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/components/presentation/alerts/).
+
+| Basic Dialog in Material 3  |  Alert in Human Interface Guidelines |
+|---|---|
+| <img alt="Basic Dialog in Material 3" src="/assets/images/docs/platform-adaptations/m3-alert.png">  |  <img alt="Alert in Human Interface Guidelines" src="/assets/images/docs/platform-adaptations/cupertino-alert.png"> |
+
+Because alert dialogs are tightly integrated with the operating system 
+and users have been trained to recognize and respond to them, 
+it's recommended to follow the platform conventions. 
+Specifically, you can use Flutter’s `AlertDialog` widget on Android 
+and the `CupertinoAlertDialog` widget on iOS. 
+Here’s a [code sample](https://gist.github.com/InMatrix/1a61ba065c0c36ae642ff934971ab97b) 
+you can use to achieve the recommended adaptation behavior. 
+Further details about adapting alert dialogs is available in 
+[this article](https://github.com/InMatrix/veggieseasons_adaptive/discussions/15).
```

Comment:
```suggestion
Further details about adapting alert dialogs is available in
this [multiplatform version of the VeggieSeasons example][veggie-seasons].

[veggie-seasons]: https://github.com/InMatrix/veggieseasons_adaptive/discussions/15.
```

Reply:
The URL points to a page more specific than the suggested label. Would something like "...available in this [preliminary alert dialog adaptation guide][alert-guide]" work?

Reply:
Yes, that would be fine.